src/post/header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PublishedDate from './publishedDate'
+import UserId from './userId'
```

Comment:
Rename the `publishedDate.js` and `userId.js` files to `PublishedDate.js` and `UserID.js`
Change `./publishedDate` and `./userId` to `./PublishedDate` and `./UserID`

React components should follow Pascal case, not camel case.
[Read this.](https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case-in-computer-programming#:~:targetText=Pascal%20case%20is%20a%20subset,UserAccount%20is%20a%20Pascal%20case.&targetText=Pascal%20is%20a%20proper%20noun,not%20capitalize%20the%20first%20letter.)